### ISSUE TITLE
Support for readonly input elements

### ISSUE THREAD

**Issue by mpryvkin** [istvan-ujjmeszaros/bootstrap-touchspin#97]
It would be nice to have support for `readonly` input element.

This could be done by changing
```
if (originalinput.is(':disabled')) {
   return;
}
```
to
```
if (originalinput.is(':disabled,[readonly]')) {
   return;
}
```

**Comment by fr0z3nfyr** [comment]
Wondering if the suggested solution is enough and won't have undesired side effects. I'd love to have the support for `readonly` prop too

**Comment by istvan-ujjmeszaros** [comment]
This is fixed in [v4.5.2](https://github.com/istvan-ujjmeszaros/bootstrap-touchspin/releases/tag/v4.5.2). FYI the buttons are being disabled when the input is readonly to avoid any confusion.

**Comment by mpryvkin** [comment]
@istvan-ujjmeszaros, thank you for your reply. Try using keys UP and DOWN in a `readonly` element, the event handlers do not check for `readonly` property and modify the value of the element.

**Comment by istvan-ujjmeszaros** [comment]
Thanks @mpryvkin, the issue with the UP/DOWN keys and the mouse wheel on readonly inputs is fixed in [v4.6.0](https://github.com/istvan-ujjmeszaros/bootstrap-touchspin/releases/tag/v4.6.0)
